src/v2/Components/ArtworkFilter/index.tsx

```diff
@@ -317,7 +324,7 @@ export const BaseArtworkFilter: React.FC<
       {/* Desktop Artwork Filter */}
       <Media greaterThan="xs">
         {tokens.version === "v3" && (
-          <Flex justifyContent="space-between" alignItems="flex-start" pb={4}>
+          <Flex justifyContent="space-between" alignItems="flex-start" pb={40}>
```

Comment:
**question**: What prompted the change from 4 to 40 here? Should we try to keep to our defined design system spacing options?

Reply:
That wasn't intentional, I was just playing around with layout styles and accidentally changed it😅